app/views/facility_orders/show.html.haml

```diff
@@ -27,7 +27,9 @@
   = render "order_table", order_details: @order_details, cross_core: false, facility_id: current_facility.id
   = render "order_table_footer", order: @order, cross_core: false
 
-%h2= t("views.facility_orders.show.cross_core_orders_header")
+- subheader_key = (@order.cross_core_project_id.present? ? "views.facility_orders.show.add_to_cross_core_order_header" : "views.facility_orders.show.add_to_order_header")
```

Comment:
nit: `@cross_core_project` is already set and used a little bit above. Can you update this to use that var?